README.md

```diff
@@ -149,6 +156,13 @@ None right now.
 - **How to sign up**: Make a PR on the [lakeFS Repo](https://github.com/treeverse/lakeFS) and we'll send you a link.
 - **Notes**: Check out our [contributing guide](https://docs.lakefs.io/contributing) and join our [slack channel](https://join.slack.com/t/lakefs/shared_invite/zt-g86mkroy-186GzaxR4xOar1i1Us0bzw) for help and discussions
 
+#### **CircleCI** (T-shirt,stickers and Magic Pin)
+
+- **Requirements**: 1-3 PRs for stickers,4 PRs for a T-Shirt and All the items with the pin if you develop a new Orb from scratch
+- **Swag**: T-shirt, stickers
+- **How to sign up**: Make the contrbutions accordingly and then use [this](https://circleci-community.typeform.com/to/mErGL0nv) form to submit your PRs
+- **Notes**: Join the #Orbtoberfest discussion on our [community forum](https://discuss.circleci.com/t/official-hacktoberfest-and-orbtoberfest-thread/37510) for ideas, FAQs, and to ask any questions. Also check out the official website (here)[https://hacktoberfest.circleci.com/]
```

Comment:
Take another peak at the last link in this line, please. You have reversed the link syntax. 😄 It should be ```[here](URL)```
```suggestion
- **Notes**: Join the #Orbtoberfest discussion on our [community forum](https://discuss.circleci.com/t/official-hacktoberfest-and-orbtoberfest-thread/37510) for ideas, FAQs, and to ask any questions. Also check out the official website [here](https://hacktoberfest.circleci.com/)
```